_posts/2018-04-21-welcome-to-this-week-in-rust.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Welcome to this week in Rust-WASM! 001"
+date:   2018-04-21 21:15:00 +0200
+---
+Hello and welcome to the another issue of *This Week in Rust+WASM*!
+
+[Rust](http://rust-lang.org) is a systems language pursuing the trifecta: safety, concurrency, and speed.
+
+[WASM](http://webassembly.org) is designed as a portable target for compilation of high-level languages like C/C++/Rust, enabling deployment on the web for client and server applications.
+
+This is a weekly summary of its progress and community.
+
+Tweet us at [@rustwasm](https://twitter.com/rustwasm) or send us a pull request. Want to get involved? [We love contributions](https://github.com/rust-lang-nursery/rust-wasm/blob/master/README.md)
+
+## News
+* [WebAssembly Studio - Online IDE](https://hacks.mozilla.org/2018/04/sneak-peek-at-webassembly-studio/)
+* [Game of Life - Build your game with WebAssembly](https://rust-lang-nursery.github.io/rust-wasm/game-of-life/introduction.html)
+* [Svelte -> Twiggy (A code size profiler for WASM written in Rust)](https://github.com/rustwasm/twiggy)
+    - [Analyze bloat from monomorphisation of generic functions](https://github.com/rustwasm/twiggy/pull/40)
+* [GGez on Rust-WASM](https://wiki.alopex.li/GgezOnWasm)
+
+
+## Cool tools to watch out for
+* [webchat-client-rs](https://github.com/bzar/webchat-rs)
+    - A doodle to create a mostly-rust client-server web app with a shared data model and binary communication over websocket between a server and WebAssembly client built using wasm-bindgen.
+
+## Blog posts
+* [NES-Emulator in Rust-WASM](https://medium.com/@bokuweb17/writing-an-nes-emulator-with-rust-and-webassembly-d64de101c49d)
+* [Hello WASM-pack!](https://hacks.mozilla.org/2018/04/hello-wasm-pack/)
+* [Small WASM binaries with Emscripten](https://kripken.github.io/blog/binaryen/2018/04/18/rust-emscripten.html)
+
+
+## Crates so far
+* [wasm-snip 0.1.1 released](https://crates.io/crates/wasm-snip/0.1.1)
```

Comment:
I actually released 0.1.2 yesterday as well, so we can update this version number :)